src/renderers/index.ts

```diff
@@ -31,7 +32,8 @@ export
 class HTMLWidget extends Widget {
   constructor(html: string) {
     super();
-    this.node.innerHTML = html;
+    let range = document.createRange();
+    this.node.appendChild(range.createContextualFragment(html));
```

Comment:
As in, `createContextualFragment` was not supported, or not implemented properly?